src/app/app.component.scss

```diff
@@ -9,23 +9,131 @@
-                mat-nav-list {
-                    align-self: flex-start;
-                    button {
-                        width: 100%;
-                    }
-                }
-            }
 
-            mat-drawer {
+            .drawer {
+                position: fixed;
+                z-index: 2;
+                height: 100%;
+                top: 68px;
+                left: 0;
+                right: 0;
                 width: 310px;
                 background: $text-color-black;
+                padding-top: 22px;
+
+                .accordion-block {
+                    .accordion {
+                        cursor: pointer;
+                        text-align: left;
+                        outline: none;
+                        font-size: 15px;
+                        transition: 0.4s;
+                        width: 262px;
+                        height: 50px;
+                        background: #191919;
+                        margin: 10px 24px;
+                        border-width: 2px 0;
+                        border-style: solid;
+                        border-color: $accordion-border;
+                        border-radius: 10px;
+                        align-items: center;
+                        display: flex;
+
+                        .left-side-accordion {
+                            border-radius: 50%;
+                            margin: 9px 15px 9px 10px;
+                            background: rgba(255, 255, 255, 0.05);
+                        }
+
+                        .accordion-btn {
+                            background: #191919;
+                            border: none;
+                            color: white;
+                            cursor: pointer;
+                        }
+                    }
+
+                    .accordion:hover {
+                        color: #ffd64d;
+                        border-width: 2px 0;
+                        border-style: solid;
+                        border-color: $yellow-bg;
+                        border-radius: 10px;
+
+                        &.accordion:after {
+                            content: url('~src/assets/img/chevron_right_24px.svg');
+                        }
+
+                        .left-side-accordion {
+                            color: $yellow-bg;
+                        }
+
+                        .accordion-btn {
+                            color: $yellow-bg;
+                        }
+                    }
+
+                    .accordion:after {
+                        content: url('~src/assets/img/chevron_right_24px white.svg');
+                        margin-left: auto;
+                        margin-right: 19px;
+                    }
+
+                    .active {
+                        &.accordion {
+                            border-width: 2px 0 0 0;
+                            transition: 0ms;
+                            border-color: $yellow-bg;
```

Comment:
We using all these variables via default scss variables. Please look how it done in corporate site.

Reply:
please check all cases